src/style.css

```diff
@@ -0,0 +1,36 @@
+html {
+  background-color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+#qa-block {
+  margin-top: 20px;
+}
```

Comment:
Don't add styles using id here. You can do same thing by adding padding to `body`